data/repos.yaml

```diff
@@ -466,17 +466,35 @@ ANY_REGION: &ANY_REGION
     RHEL_7.6:
         <<: *RHEL_7_1
         
-    RHEL_7.7:
-        <<: *RHEL_7_1
+    RHEL_7.7: &RHEL_7_7
+        rhui-client-config-server-7: true
+        rhui-rhel-7-server-rhui-debug-rpms: false
+        rhui-rhel-7-server-rhui-extras-debug-rpms: false
+        rhui-rhel-7-server-rhui-extras-rpms: false
+        rhui-rhel-7-server-rhui-extras-source-rpms: false
+        rhui-rhel-7-server-rhui-optional-debug-rpms: false
+        rhui-rhel-7-server-rhui-optional-rpms: false
+        rhui-rhel-7-server-rhui-optional-source-rpms: false
+        rhui-rhel-7-server-rhui-rh-common-debug-rpms: false
+        rhui-rhel-7-server-rhui-rh-common-rpms: true
+        rhui-rhel-7-server-rhui-rh-common-source-rpms: false
+        rhui-rhel-7-server-rhui-rpms: true
+        rhui-rhel-7-server-rhui-source-rpms: false
+        rhui-rhel-7-server-rhui-supplementary-debug-rpms: false
+        rhui-rhel-7-server-rhui-supplementary-rpms: false
+        rhui-rhel-7-server-rhui-supplementary-source-rpms: false
+        rhui-rhel-server-rhui-rhscl-7-debug-rpms: false
+        rhui-rhel-server-rhui-rhscl-7-rpms: false
+        rhui-rhel-server-rhui-rhscl-7-source-rpms: false
 
     RHEL_7.8:
```

Comment:
Note that AMIs created in the future will contain a newer client configuration RPM in which the RHUI repo names are different. The "rhui-" prefix won't be used with RHEL repos because there's "-rhui-" in the name already. So, e.g. rhui-rhel-7-server-rhui-rpms will become rhel-7-server-rhui-rpms. See `/etc/yum.repos.d/redhat-rhui.repo` in the latest `rh-amazon-rhui-client` package.

Reply:
Thanks, Radek
I will check and sync it in next patch.